### ISSUE TITLE
任务书的模版太凑合了

### ISSUE THREAD

**Issue by beantowel** [BHOSC/BUAAthesis#225]
内容居然是手动一行一行填写的，而且行数是固定的，不能多也不能少，插入任务资料的引用基本实现不了。

**Comment by JosephPeng** [comment]
(扶额笑)
Personally, I can't agree more, the code of that part is ... a little bit ugly.
Unfortunately, we can't find a property method to meet the requirements of the department of BUAA other than that way.
It would be appreciated if there are some more artful ways to achieve the requirements.

**Comment by sighingnow** [comment]
We do have tried to improve this page. However we found it's not easy to reach a more elegant solution for this bug.

I have submitted PR #191 to solve #189. Finally @huxuan and I both agreed that we should have a fixed style in these two pages and every part should occupy same lines whether there is enough contents.

> From user's perspective, break the line manually may be a little ugly, but it works for having a fixed style of corresponding part. It means even when we have just a little or even no content, they should still occupy same number of lines.

See more related discussion in #191. For expert users, see [this diff](https://github.com/BHOSC/BUAAthesis/pull/191/files#diff-01ab0d17b918ee69b08eb3e1d6c910a0L775) for how to redefine the `\ulinel` command properly to remedy the trouble.
